doc/classes/@GlobalScope.xml

```diff
@@ -2293,16 +2294,16 @@
 			Middle mouse button.
 		</constant>
 		<constant name="MOUSE_BUTTON_WHEEL_UP" value="4" enum="MouseButton">
-			Mouse wheel up.
+			Mouse wheel scrolling up.
 		</constant>
 		<constant name="MOUSE_BUTTON_WHEEL_DOWN" value="5" enum="MouseButton">
-			Mouse wheel down.
+			Mouse wheel scrolling down.
 		</constant>
 		<constant name="MOUSE_BUTTON_WHEEL_LEFT" value="6" enum="MouseButton">
-			Mouse wheel left button (only present on some mice).
+			Mouse wheel scrolling left (only present on some mice).
```

Comment:
Technically, on the mice I used, this is and actually feels like a button. It has only one state: pressed and released.

The up/down wheel has multiple discrete states so it's different.